src/telemetry/telemetryService.ts

```diff
@@ -130,6 +130,10 @@ export type CopilotIntroductionProperties = {
   is_copilot_active: boolean;
 };
 
+export type ParticipantOpenedFromInputBoxProperties = {
+  source?: 'query with copilot codelens';
```

Comment:
We have the `DOCUMENT_SOURCE` enum which maybe could be relevant here but this seems quite different?

Reply:
Do we use `source` only for telemetry or how else is this used?

Reply:
Yep, just telemetry. Specifically would be useful for Gaurab to track how much query with copilot is used. I thought it'd be better to do it like this instead of introducing a new event just for query with copilot